tests/foreman/ui/test_role.py

```diff
@@ -660,113 +654,156 @@ def test_positive_disable_all_filters_overriding_option(self):
+            2. Create user with different taxonomies than above org admin
+                taxonomies
+            3. Assign above cloned role to user
+            4. Login with user and attempt to access resources
+
+        :expectedresults: User should not be able to access any resources and
+            permissions in taxonomies selected in org admin role
+        """
+
+    @stubbed
+    @tier3
+    def test_negative_access_entities_from_user(self):
+        """User can not access resources within its own taxonomies if assigned
+            role does not have permissions for same taxonomies
+
+        :id: ec5fc7d9-0150-465f-b5b0-ff21d75b8e8e
+
+        :steps:
+
+            1. Create org_admin and assign taxonomies to it
+            2. Create user with different taxonomies than above org admin
+                taxonomies
+            3. Assign above cloned role to user.
+            4. Login with user and attempt to access resources
+
+        :expectedresults: User shoule not be able to access any resources and
+            permissions in his own taxonomies
+        """
+
+    @stubbed
+    @tier2
     def test_positive_override_cloned_role_filter(self):
-        """Test if the cloned role filter can be overrided
+        """Cloned role filter overrides
 
         :id: e475aa7d-4844-4bb3-bfd3-3d4082a41fe4
 
         :steps:
 
-            1. Clone any existing role(e.g Manager Role)
-            2. Attempt to override the filter in cloned role
+            1. Create a role with overridable filter.
+            2. Clone above role.
+            3. Attempt to override the filter in cloned role
 
-        :expectedresults: Filter in cloned role should be successfully
-            overriding
+        :expectedresults: Filter in cloned role should be overriding
 
         :caseautomation: notautomated
 
         :CaseImportance: Critical
         """
 
     @stubbed
-    @tier1
-    def test_positive_empty_filter_taxonomies_in_cloned_role(self):
-        """Test overridden filters taxonomies in cloned role
+    @tier2
+    def test_positive_emptiness_of_filter_taxonomies_on_role_clone(self):
+        """Taxonomies of filters in cloned role are set to None for filters that
+            are overridden in parent role
 
         :id: 86d9cd93-8189-45b3-86c6-ac9185e48655
 
-        :steps: Clone a role having overridden filter(s), where filters should
-            have some taxonomies (locations and organizations) assigned
+        :steps:
+
+            1. Create a role with an overridden filter
+            2. Overridden filter should have taxonomies assigned
+            3. Clone above role
+            4. View cloned role filters
 
         :expectedresults:
 
-            1. On cloning, taxonomies of the overridden filters in cloned role
-                are set to None
+            1. Taxonomies of the 'parent role overridden filters' are set to
+                None in cloned role
```

Comment:
I didnt understand this but make test-docstrings doesnt throw any exception for this